simulator/registry.go

```diff
@@ -114,7 +114,10 @@ func typeName(item mo.Reference) string {
 // valuePrefix returns the value name prefix of a given object
 func valuePrefix(typeName string) string {
 	if v, ok := refValueMap[typeName]; ok {
-		return v
```

Comment:
Should we just return `v` here? As the prefix check targets "domain-c".

Reply:
We only want to return `v` if it already contains a `-` (domain-c being the only case currently). Otherwise we want to append a `-`.